packages/nimble-components/src/toggle-tip/specs/toggle-tip-ixd.md

```diff
@@ -0,0 +1,147 @@
+
+**When to use:**
+
+-   In a _form_ or _editor_ view to display additional information or context for a specific element.
+-   When the additional information or context provided by the toggletip is not valuable for most users of the control.
+
+**When not to use:**
+
+-   When the additional information or context is valuable for most users of the control. This context should be visible by default.
+
+## Anatomy
+
+![Toggletip anatomy](./spec-images/anatomy.png)
+
+| Element        | Description                                                                                        |
+| -------------- | -------------------------------------------------------------------------------------------------- |
+| Trigger button | Info icon button for showing and hiding an associated popover                                      |
+| Popover        | Overlay container for text and interactive components                                              |
+| Content area   | Area consisting of slots for a title, 1-3 lines of text, and a button or link. See the Banner API. |
+
+## Sizing
+
+![Minimal toggletip](./spec-images/minimal-toggle-tip.png)
+
+![Maximal toggletip](./spec-images/maximal-toggle-tip.png)
+
+### Width
+
+The width of the toggletip popover should be determined based on the length of the content it contains. However, if the content is greater than a single line of text, the width should be set to 300px.
+
+### Height
+
+When the popover height exceeds that of the viewport, the popover should not become scrollable to try and fit in the viewport (not pictured). The popover should always be tall enough to show all its content. The surrounding content, for example the page itself, should scroll instead.
+
+Popovers with text should be limited to approximately three lines of text under normal display size at the intended location.
+
+## Alignment
+
+The toggletip popover should avoid obscuring other content where possible. [^1] In particular it should avoid obscuring the content which it annotates.
+
+[^1]: [Pearson Toggletip requirements](https://accessibility.pearson.com/resources/developers-corner/reference-library/tooltips-and-toggletips/index.php)
+
+![Popover alignment](./spec-images/popover-orientation.png)
+
+The toggletip popover should be aligned with the "Top - Right" of trigger button by default. If there isn't enough room in the viewport for the entire popover to be visible, the popover should align with the trigger button in the following order:
+
+1. Top - Right
+2. Top - Left
+3. Bottom - Right
+4. Bottom - Left
+
+## Behavior
+
+### States
+
+![Toggletip states](./spec-images/states.png)
+
+#### Error State
+
+The toggle-tip does not support an error state. Further, the toggle-tip should not be used to display error information.
+
+### ARIA Considerations
+
+> NOTE: From Pearson's [Toggletip](https://accessibility.pearson.com/resources/developers-corner/reference-library/tooltips-and-toggletips/index.php) reference document.
+
```

Comment:
Maybe we should have a note or something saying that the following two sections should follow the intent / principles discussed by the aria from the Pearson's toggletip but the implementation should be defined in terms of nimble concepts.

Nimble does some things differently than discussed but towards the same ends so some of the "MUST" language should be taken in that context.